frontend/src/static/js/main.js

```diff
@@ -223,7 +234,16 @@ window.app = new window.Vue({
 
       if (this.isTabActive) {
         if (hash.tabType === 'authorship') {
-          this.renderAuthorShipTabHash(hash.since, hash.until);
+          let { since, until } = hash;
+
+          // get since and until dates from window.app if not found in hash
+          if (!since) {
+            since = window.app.sinceDate;
```

Comment:
perhaps can shorten to something like
`since = hash.since ? hash.since : window.app.sinceDate;`?

Reply:
Updated accordingly.